packages/fxa-content-server/app/scripts/models/user.js

```diff
@@ -21,14 +21,31 @@ const ResumeTokenMixin = require('./mixins/resume-token');
 const UrlMixin = require('./mixins/url');
 const Storage = require('../lib/storage');
 const vat = require('../lib/vat');
+const ProfileClient = require('../lib/profile-client');
+const MarketingEmailClient = require('../lib/marketing-email-client');
+const OAuthClient = require('../lib/oauth-client');
+const InterTabChannel = require('../lib/channels/inter-tab');
+const FxaClient = require('../lib/fxa-client');
 
 var User = Backbone.Model.extend({
   initialize (options = {}) {
+    this._config = options.config;
```

Comment:
`config` is not passed in when creating the [User object in app-start.js](https://github.com/hritvi/fxa/blob/d4d06d5b48ef0d79e60af6aa23e0311bee516e86/packages/fxa-content-server/app/scripts/lib/app-start.js#L397), fetching the URLs from config will fail because it'll be undefined. You'll need to pass in `config` when creating the object to avoid the failure.

Reply:
Oh right!